containers/src/Kokkos_UnorderedMap.hpp

```diff
@@ -286,7 +286,8 @@ class UnorderedMap {
 
   enum { modified_idx = 0, erasable_idx = 1, failed_insert_idx = 2 };
   enum { num_scalars = 3 };
-  using scalars_view = View<int[num_scalars], LayoutLeft, device_type>;
+  using scalars_view     = View<int[num_scalars], LayoutLeft, device_type>;
+  using scalar_host_view = View<size_type[1], HostSpace>;
```

Comment:
I would use a zero-dimensional View instead since we are really wrapping a singular rather than a one-dimensional range consisting of one element.
```suggestion
  using scalar_host_view = View<size_type, HostSpace>;
```